src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -105,6 +105,9 @@ export interface IWebXRControllerPointerSelectionOptions {
      */
     maxPointerDistance?: number;
 
+    /**
+     * Disable near interaction if not used to improve performance.

```

Comment:
nit: punctuation, or it'll sound like one of the uses for near interaction is to _improve_ performance